src/App.js

```diff
@@ -1,45 +1,39 @@
-import React, { useState } from 'react';
+import React from 'react';
+import {
+  BrowserRouter as Router, Route, Link, Routes,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
 import HomePage from './components/HomePage';
 import FunnyQuot from './components/FunnyQuot';
 import './components/calculator.css';
 
```

Comment:
![image](https://github.com/jubaer919/math-magicians/assets/93039359/29dd7fff-0aa7-4560-b956-31299c8e279e)
- [ ] It would be nice if you add some meaningful features in your Future Feature section to make your readme descriptive and professional 👍